src/man/sssd.conf.5.xml

```diff
@@ -790,6 +790,32 @@
                         </para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term>refresh_max_random_offset (integer)</term>
+                    <listitem>
+                        <para>
+                            When SSSD is in offline mode it keeps probing
+                            backend servers in specified time intervals:
+                        </para>
+                        <para>
+                            delay = delay + (rand() % random_offset)
+                        </para>
+                        <para>
+                            This parameter controls the value of the random offset
+                            used for above equasion. Final random_offset value
+                            will be random number from range:
```

Comment:
I think it should be written `in range` but best to check with a native speaker.

Reply:
Fixed